skt/kernelbuilder.py

```diff
@@ -97,7 +98,7 @@ def make_redhat_config(self):
         # Copy the correct kernel config into place
         # TODO Add multi-arch support later.
         config_files = glob.glob(
-            "{}/redhat/configs/kernel*x86_64.config".format(self.source_dir)
+            "{}/configs/kernel*x86_64.config".format(self.source_dir)
```

Comment:
This doesn't seem related to renaming the function. Please do this in another, appropriate commit.